homeassistant/components/light/__init__.py

```diff
@@ -43,6 +43,9 @@
    A list containing three integers representing the xy color you want the
    light to be.
 
+ - ct_color
+   An INT in mireds represending the color temperature you want the light to be
+
```

Comment:
As a suggestion; you might want to specify the CT in Kelvin since it is more known in the normal world. Usually color temperature range on packaging/tech specs is listed in Kelvin. 

Kelvin to Mired conversion is trivial see https://en.wikipedia.org/wiki/Mired